### ISSUE TITLE
Update install

### ISSUE THREAD

**Issue by RichardLitt** [simonv3/reading-list#8]
This suggests that I use hoodie-cli, which, according to hoodiehq/hoodie-cli, is now deprecated. See https://github.com/simonv3/reading-list/pull/7. Should we update this app?

**Comment by simonv3** [comment]
Yeah, Hoodie went through a big re-write since I wrote this. It's possible that it doesn't work anymore in the same way.

**Comment by simonv3** [comment]
TBH at one point I think I started building this in Meteor https://github.com/simonv3/reading-list-meteor with the intention of having it be a Sandstorm app. But I don't remember how far I got with that.

**Comment by RichardLitt** [comment]
Cool. Are you going to keep it up?

**Comment by simonv3** [comment]
Keep what up? This repo? Is there a reason I should take it down?

**Comment by RichardLitt** [comment]
Keep up development. If not, add a 'not currently active' note in the README or the description, maybe? 

This is linked on your Patreon, which is how I found it.

**Comment by simonv3** [comment]
You're more than welcome to work on it!
